daemon/team.go

```diff
@@ -2,11 +2,20 @@ package daemon
 
 import (
 	"context"
+	"math"
+	"time"
+
+	"github.com/aau-network-security/haaukins/virtual"
+
 	pb "github.com/aau-network-security/haaukins/daemon/proto"
 	"github.com/aau-network-security/haaukins/store"
 	"github.com/rs/zerolog/log"
 )
 
+const (
+	INACTIVITY_DURATION = 8 // in hours
```

Comment:
lets document it somewhere or pass this value from config file.

Reply:
yup.